### PR TITLE
join cert to single line fix getlantern/lantern#3915

### PR DESCRIPTION
**Pull request by fffw**

### CODE REVIEW

File: salt/http_proxy/save_access_data.py [hunk 1]
```diff
@@ -44,7 +46,7 @@ def add_obfs4_access_data(d_out):
             if m:
                 d_out['cert'] = m.group(1)
```
Comment (fffw) [inline salt/http_proxy/save_access_data.py]: The cert of `obfs4` should be in single line as they are matched using `regex`.

**Comment by aranhoide** [comment]
LGTM!  Have you tested launching a new server with this config?

**Comment by fffw** [comment]
@aranhoide No I'm not that familiar with the process to launch a new server. I verified only by executing that Python snippet directly.

**Comment by aranhoide** [comment]
OK, I'm trying it; I'll document the process to launch a new server tomorrow morning.

**Comment by aranhoide** [comment]
@fffw: I created a new test proxy (188.166.38.120) with this configuration, and for some reason it's failing fallback checks.

I'm exhausted now; I'll look into this tomorrow morning.

**Comment by aranhoide** [comment]
A variation of this that just normalizes on single '\n's passes checkfallbacks and the uploaded cert looks right, so I'll push this.

**Comment by fffw** [comment]
Cool thanks so much @aranhoide

**Pull request merged**